packages/manager/apps/pci/package.json

```diff
@@ -27,6 +27,8 @@
     "@ovh-ux/ng-ovh-responsive-popover": "^5.0.0-beta.0",
     "@ovh-ux/ng-ovh-swimming-poll": "^4.0.0",
     "@ovh-ux/ng-ovh-user-pref": "^1.0.0",
+    "@ovh-ux/ng-pagination-front": "^8.0.0-alpha.0",
```

Comment:
I think both `"@ovh-ux/ng-pagination-front"` and `"@ovh-ux/ng-q-allsettled"` has been bumped into a new major release due to the switch to a rollup stack configuration.

In my opinion, if these dependencies pass the validation for the related Control Panel, we should release a final major version in order to not have `alpha/beta` dependency in production environment.

Do you agree?